util/archutil/Dockerfile

```diff
@@ -8,7 +8,9 @@ RUN apt-get update && apt-get --no-install-recommends install -y \
   binutils-s390x-linux-gnu \
   binutils-powerpc64le-linux-gnu \
   binutils-mips64el-linux-gnuabi64 \
-  binutils-mips64-linux-gnuabi64
+  binutils-mips64-linux-gnuabi64 \
+  binutils-loongarch64-linux-gnu \
```

Comment:
```
4.151 E: Unable to locate package binutils-loongarch64-linux-gnu
```

Reply:
Only `amd64`, `i386`, not `aarch64` :cry:: https://packages.debian.org/testing/binutils-loongarch64-linux-gnu